### PR TITLE
[Front-End] Book Add CSS and Functionality Changes

### PR DESCRIPTION
**Pull request by nex-pixel**
## Feature Description/Implementation
Changed sizing of the book add page edit table, added functionality for deleting individual line items, and clearing the table, and adding more new books with a simple lookup.

### CODE REVIEW

File: frontend/src/pages/books/BookAdd.tsx [hunk 1]
```diff
@@ -280,6 +281,9 @@ export default function BookAdd() {
     BOOKS_API.addBookInitialLookup({ isbns: textBox })
       .then((response) => {
         setIsLoadingButton(false);
+
+        const _data = [...books];
+        setBooks(_data);
```
Comment (caseyszilagyi) [inline frontend/src/pages/books/BookAdd.tsx:286]: Why was this added? Don't we already set all of the books in the loop below?

**Review by caseyszilagyi — CHANGES_REQUESTED** [review]

File: frontend/src/pages/books/BookAdd.tsx (hunk 1 above)
Comment (nex-pixel) [reply frontend/src/pages/books/BookAdd.tsx:286]: This was added to have the funtionality where clicking lookup didn't erase the currently filled in set of data for the edit book fields data. That way the user has the flexibility to delete individual line items from earlier lookups, and add more books to the edit fields table with additional lookup requests.

**Review by caseyszilagyi — APPROVED** [review]
